documentation/reference/query-language-reference.html

```diff
@@ -724,6 +724,71 @@ <h2 id="where">where</h2>
     </tbody>
     </table>
   </td></tr>
+
+  <tr id="nearestneighbor"><th>nearestNeighbor</th><td>
+    <p>
+    <em>nearestNeighbor</em> matches the top-k nearest neighbors in a multi-dimensional vector space.
+    Points in the vector space are specified as
+    <a href="../tensor-user-guide.html">tensors</a> with one indexed dimension,
+    where the size of that dimension is equal to the dimensionality of the vector space.
+    The document positions are stored in a tensor attribute, and the query position is sent with the query request.
+    Euclidean distance is used as the distance function and the exact nearest neighbors are returned.
+    Example:
+
+<pre class="urlunencode" oncopy="">
+where%20%5B%7B%22targetNumHits%22%3A%2010%7D%5D%20nearestNeighbor%28doc_vector%2C%20query_vector%29%3B&amp;ranking.features.query%28query_vector%29=%5B3%2C5%2C7%5D
+</pre>
+    </p>
+    <p>
+    In this example we search for the top 10 nearest neighbors in a 3-dimensional vector space.
+    <em>targetNumHits</em> specifies the wanted top-k nearest neighbors to find. This parameter is required.
+    The first parameter of <em>nearestNeighbor</em> is the name of the tensor attribute containing the document positions (<em>doc_vector</em>).
+    The second parameter is the name of the tensor sent with the query request (<em>query_vector</em>).
```

Comment:
consider copying from javadoc:
specifying "myvector" as the name means the query must set "ranking.features.query(myvector)"

Reply:
Fixed